tests/testreactive.py

```diff
@@ -433,6 +435,96 @@ def test_reactive_when_initial():
     p.param.trigger('event')
     assert integer.rx.value == 4
 
+def test_reactive_resolve():
+    p = Parameters(integer=3)
+    p2 = Parameters(parameter=p.param.integer)
+
+    prx = p2.param.parameter.rx()
+    assert prx.rx.value is p.param.integer
```

Comment:
Maybe it is better to use something other than numbers here. This will always be true if both results are 3.

Reply:
`p.param.integer` is a Parameter object, not a number.